class.jetpack-gutenberg.php

```diff
@@ -303,7 +303,10 @@ public static function preset_exists( $preset ) {
 	 * @return mixed Returns an object if the file is present, or false if a valid .json file is not present.
 	 */
 	public static function get_preset( $preset ) {
-		return json_decode( file_get_contents( JETPACK__PLUGIN_DIR . self::get_blocks_directory() . $preset . '.json' ) );
+		return json_decode(
+			// //phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
```

Comment:
nit-pick: double set of `//`

Reply:
Doh. Will fix.

Reply:
502f313e06c8c4f720f430c871ad081491a7485d